ceph/ceph_admin/bootstrap.py

```diff
@@ -182,7 +182,7 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
         if custom_repo:
```

Comment:
can we have this from test config and applying definition to figure out what has to be set in `ceph_admin.py`.
ex., 
for custom repo, `custom_repo: "http://path_to.com/tools/repo"`
for CDN ,`custom_repo: None `(in fact we could not define this variable in test config) 

we can have discussion other possibilities.